demos/serverless/template.yaml

```diff
@@ -7,6 +7,15 @@ Parameters:
     Default: false
     Type: String
     AllowedValues: [true, false]
+  Region:
+    Description: The AWS SDK Chime REGION
+    Default: "us-east-1"
+    Type: String
+  UseChimeMeetingsSDK:
+    Description: Use boolean to switch between Chime and ChimeSDKMeetings Client
```

Comment:
This says ChimeSDKMeetings and few other places say ChimeMeetingsSDK, can you check and make this consistent with the actual namespace: `ChimeSDKMeetings` in other places.